packages/databricks-sdk-js/src/services/wsfs/WorkspaceFsDir.ts

```diff
@@ -95,7 +95,7 @@ export class WorkspaceFsDir extends WorkspaceFsEntity {
                 {
                     path: validPath,
                     overwrite,
-                    format: "AUTO",
```

Comment:
We need it to be auto to correctly import notebooks. I added it manually. If you are already updating it, maybe you can add the type to openapi as well?

Reply:
reverted that change